core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -352,6 +352,16 @@ private boolean checkVersionConflict(
             final long currentVersion,
             final long expectedVersion,
             final boolean deleted) {
+        if (op.versionType() == VersionType.FORCE) {
+            if (engineConfig.getIndexSettings().getIndexVersionCreated().after(Version.V_5_0_0_alpha1)) {
```

Comment:
Actually, now that I think about this, I don't want to make this change. If someone ignores our advice and put 5.0.0-alpha1 into production, we still want to disallow `force` for those indices